Analysis/include/EventTags.h

```diff
@@ -21,25 +21,36 @@ class EventTagCreator {
     using DataId = EventTags::DataId;
 
     EventTagCreator(const EventCategorySet& _categories, const EventSubCategorySet& _subCategories,
-                    const std::map<SelectionCut, analysis::EllipseParameters>& _massWindowParams,
-                    const std::set<UncertaintySource>& _event_unc_sources,
-                    const std::set<UncertaintySource>& _norm_unc_sources,
-                    bool _use_kinFit, bool _use_svFit);
+                     const std::map<SelectionCut, analysis::EllipseParameters>& _massWindowParams,
+                     const std::set<UncertaintySource>& _event_unc_sources,
+                     const std::set<UncertaintySource>& _norm_unc_sources,
+                     bool _use_kinFit, bool _use_svFit);
```

Comment:
- Add flag whatever to use iterative fit btag SFs or combined ones.
- remove use_kinFit and use_svFit